python/ray/tests/test_cli.py

```diff
@@ -420,7 +430,15 @@ def commands_mock(command, stdin):
         print("This is a test!")
         return PopenBehaviour(stdout=b"This is a test!")
 
-    with _setup_popen_mock(commands_mock):
+    def commands_verifier(calls):
+        for call in calls:
+            if len(call[1]) > 0:
+                for _ in call[1][0]:
+                    if " ray stop; " in _:
+                        return True
```

Comment:
nit: `if any(" ray stop; " in token for token in call[1][0])`

Reply:
Thanks!